packages/dd-trace/src/appsec/telemetry.js

```diff
@@ -89,7 +89,12 @@ function updateRaspRequestsMetricTags (metrics, req, raspRuleType) {
 
   if (!enabled) return
 
-  const tags = { rule_type: raspRuleType, waf_version: metrics.wafVersion }
+  const tags = { rule_type: raspRule.type, waf_version: metrics.wafVersion }
+
+  if (raspRule.variant) {
+    tags.rule_variant = raspRule.variant
```

Comment:
for the record: Metrics and its tags must be defined [here](https://github.com/DataDog/dd-go/blame/prod/trace/apps/tracer-telemetry-intake/telemetry-metrics/static/common_metrics.json#L804)

And `rasp.rule.eval` has `rule_variant` tag, so perfect! 
it's very common to forget ;)